### ISSUE TITLE
Input validation sniff doesn't support multiple vars per isset() check

### ISSUE THREAD

**Issue by JDGrimes** [WordPress-Coding-Standards/WordPress-Coding-Standards#183]
This will get flagged for not being validated, but shouldn't:

```php
if ( isset( $_POST['some_var'], $_POST['another_var'] ) ) {
      $some_var = $_POST['some_var'];
      $another_var = $_POST['another_var']; // This will be flagged for no validation
}
```

**Comment by JDGrimes** [comment]
This is fixed in 0.3.0.
